style.css

```diff
@@ -26,9 +26,63 @@ form {
 }
 
 input {
-  margin: 10px;
+  width: 90%;
+  margin: 15px;
+  height: 30px;
+  padding: 3px;
+  border: 2px solid rgb(61, 60, 60);
 }
 
 #addBtn {
   width: 100px;
+  margin-left: auto;
+  font-weight: 700;
+  border-radius: 4px;
+  box-shadow: 5px 5px rgb(61, 60, 60);
+}
+
+.row-odd {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  height: 40px;
+  background-color: rgb(255, 255, 255);
+  padding: 3px;
+}
+
+.row-even {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 3px;
+  height: 40px;
+  background-color: rgb(172, 172, 172);
+}
+
+.remove {
+  height: 25px;
+  width: 70px;
+}
+
+#bookList {
+  margin: 50px 0;
+  border: 2px solid rgb(61, 60, 60);
+}
+
+hr {
+  width: 40%;
+  border-color: rgb(61, 60, 60);
+}
+
+#addBooks {
+  margin-top: 35px;
+}
+
+#newBook {
+  display: flex;
```

Comment:
- [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```